app/src/main/java/edu/kmaooad/capstone23/members/commands/CreateMemberBasic.java

```diff
@@ -0,0 +1,52 @@
+package edu.kmaooad.capstone23.members.commands;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+
+public class CreateMemberBasic {
```

Comment:
If this command creates basic members, it should be named CreateBasicMember, just as we say in normal language. We are not saying "create member basic".